wire-runtime/src/main/java/com/squareup/wire/Message.java

```diff
@@ -56,6 +56,12 @@ public final ByteString unknownFields() {
    */
   public abstract Builder newBuilder();
```

Comment:
This returns a raw type instead of a correctly parameterized one. Unfortunately to fill it in properly we'd need to add `B extends Builder<T, B>` to `Message`. Otherwise the best we can do is `Builder<T, ?>` which is unfortunate...